src/applications/disability-benefits/526EZ/tests/config/primaryAddress.unit.spec.jsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
+/*  */import React from 'react';
```

Comment:
It looks like this comment can be removed.

Reply:
Oh. Uhhhh....yup! No idea how that sneaked in there. :confused: